src/utils/supergraph_config.rs

```diff
@@ -116,28 +119,35 @@ fn merge_supergraph_configs(
     local_config: Option<SupergraphConfig>,
     target_federation_version: Option<&FederationVersion>,
 ) -> Option<SupergraphConfig> {
+    // We use the federation version explicitly given at the command line as top
+    // priority; otherwise the version explicitly given in the local config
+    // file; otherwise the version fetched from Studio.
```

Comment:
I'd add something about defaulting to latest_fed_two just to make it explicit that if there's no target, no remote, and no local, we're going to use that one